app/views/state_file/questions/eligible/edit.html.erb

```diff
@@ -38,6 +46,8 @@
         <%= render partial: "state_file/questions/eligible/az_unsupported" %>
       <% elsif current_state_code == "ny" %>
         <%= render partial: "state_file/questions/eligible/ny_unsupported" %>
+      <% elsif current_state_code == "id" %>
+        <%= render partial: "state_file/questions/eligible/id_unsupported" %>
```

Comment:
@arinchoi03 this approach brakes MD because it has disqualifying_eligibility_rules but it doesn't have that partial. should I just make a blank partial that we can edit later?

Reply:
@DrewProebstel that sounds fine to me!